### PR TITLE
Support CMS with top N

### PR DESCRIPTION
**Pull request by erjiaqing**
What did I do:

1. Add a struct to CMSketch

```proto
message CMSketchTopN {
    optional bytes data = 1;
    optional uint64 count = 2 [(gogoproto.nullable) = false];
}
```

2. Run make.

Related PR: https://github.com/pingcap/tidb/pull/10255

### CODE REVIEW

File: proto/analyze.proto [hunk 1]
```diff
@@ -106,6 +106,13 @@ message CMSketchRow {
     repeated uint32 counters = 1;
 }
 
+message CMSketchTopN {
+    optional bytes data = 1;
+    required uint64 count = 2 [(gogoproto.nullable) = false];
```
Comment (zz-jason) [inline proto/analyze.proto]: better to change it to optional?

**Comment by shenli** [comment]
@erjiaqing Please fill the description.

File: generate-go.sh [hunk 2]
```diff
@@ -1,5 +1,4 @@
-#!/bin/bash
-
+#!/usr/bin/env bash
```
Comment (lamxTyler) [inline generate-go.sh]: Why change it?

Comment (erjiaqing) [reply generate-go.sh]: It used to be /usr/bin/env bash, and I created this branch before it is changed to /bin/bash (and rebased before file pr), I will correct this.

**Comment by lamxTyler** [comment]
@erjiaqing Please resolve the conflicts.

**Pull request merged**